fail2ban/tests/utils.py

```diff
@@ -50,6 +54,10 @@
 	else:
 		CONFIG_DIR = '/etc/fail2ban'
 
+# In not installed env (setup, test-cases) use fail2ban modules from main directory:
+if 1 or os.environ.get('PYTHONPATH', None) is None:
```

Comment:
ie shouldn't it be just `if os.environ.get('PYTHONPATH'):` (not sure if empty PYTHONPATH should have some magical treatment) ?